modules/images-configuration-create-config-map.adoc

```diff
@@ -0,0 +1,61 @@
+[id="steps-for-manual-configmap-creation"]
+= Steps to create the image signature ConfigMap
+
+The following commands will create and apply the image signature ConfigMap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
```

Comment:
This sounds a bit awkward. How about something like: "Use the following solution article to determine a valid upgrade path for your cluster".
We should also mention that the following steps need to be ran for each upgrade target node in the path or explain that the steps are for doing a single upgrade of the path.